src/tic_tac_toe/game.clj

```diff
@@ -4,13 +4,24 @@
             [tic-tac-toe.referee :as referee]
             [tic-tac-toe.user-interface :as ui]))
 
+(defn share-information
+  [player game-ui board]
+  (->> {:game-ui game-ui :board board}
+       ((:needed player))))
```

Comment:
This feels really complicated. Is there a way we could simplify it? Can we make the interface for the Human and Computer the same?

Reply:
Adjusted the [interface](https://github.com/saralein/tic-tac-toe_clojure/pull/4/files#diff-954fe644a82dba16ad084dd0bdc1378cR11).